vck/src/commonTest/kotlin/at/asitplus/wallet/lib/data/rfc1950/CompressionFlagsTest.kt

```diff
@@ -0,0 +1,38 @@
+package at.asitplus.wallet.lib.data.rfc1950
+
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+
+class CompressionFlagsTest : FreeSpec({
+    "check properties" - {
+        "fcheck" - {
+            withData(
+                (0..31).map {
+                    it.toByte()
+                },
+            ) {
+                CompressionFlags(it).checkBits.value shouldBe it
+            }
+        }
+        "fdict" - {
+            withData(
+                listOf(0, 32).map {
+                    it.toByte()
+                },
+            ) {
+                CompressionFlags(it).hasDictionaryIdentifierAfterFlagByte shouldBe (it > 0)
+            }
+        }
+        "flevel" - {
+            withData(
+                (0..3).map {
```

Comment:
apparently 0..3 means something ... maybe add a link to the relevant RFC section?